packages/plugins/RSS3/src/locales/en-US.json

```diff
@@ -2,8 +2,10 @@
     "inactive_project": "Inactive Project",
     "no_activity_time": "No activity time",
     "attended": "attended",
-    "no_data": "No data",
+    "no_data": "No {{collection}} at the current address",
     "total_grants": "Total {{count}} Grants",
     "contribution": "Contribution",
-    "contribution_other": "Contributions"
+    "contribution_other": "Contributions",
+    "contributed": "contributed",
+    "to": "to"
```

Comment:
Can we use the total sentence with placeholders?

Reply:
No，because I have to add style to this word
<img width="620" alt="截屏2022-07-28 08 46 30" src="https://user-images.githubusercontent.com/97870249/181396787-6227b709-819f-4ab4-87b8-19f8d93089a9.png">